test-network-function/accesscontrol/suite.go

```diff
@@ -73,7 +73,8 @@ var _ = ginkgo.Describe(common.AccessControlTestKey, func() {
 
 //nolint:gocritic,funlen // ignore hugeParam error. Pointers to loop iterator vars are bad and `testCmd` is likely to be such.
 func runTestOnPods(env *config.TestEnvironment, testCmd testcases.BaseTestCase, testType string) {
-	ginkgo.It(fmt.Sprintf("Running Pod test : %s", testCmd.Name), func() {
+	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestHostResourceIdentifier)
```

Comment:
testCmd.Name is required here for the It name. All access-control sub tests have the same name otherwise

Reply:
fixed